frontend/src/lib/components/HTMLElementsDisplay/HTMLElementsDisplay.tsx

```diff
@@ -43,17 +46,20 @@ function Tags({
     return (
         <>
             {elements.map((element, index) => {
+                const invertedIndex = elements.length - 1 - index
+
                 return (
-                    <SelectableElement
-                        key={`${element.tag_name}-${index}`}
-                        element={element}
-                        isDeepestChild={index === elements.length - 1}
-                        onChange={(s) => (editable ? onChange(index, s) : undefined)}
-                        readonly={!editable}
-                        indent={indent(index)}
-                        highlight={highlight}
-                        parsedCSSSelector={parsedCSSSelectors[index]}
-                    />
+                    <Fade key={`${element.tag_name}-${invertedIndex}`} visible={true}>
```

Comment:
The inverted index helps ensure only the new elements fade in, while the others don't.